Mage/src/main/java/mage/abilities/common/SagaAbility.java

```diff
@@ -161,6 +219,9 @@ public boolean checkTrigger(GameEvent event, Game game) {
             return false;
         }
         int loreCounters = permanent.getCounters(game).getCount(CounterType.LORE);
+        if (readAhead && permanent.getTurnsOnBattlefield() == 0) {
+            return chapterFrom.getNumber() == loreCounters;
```

Comment:
I think this doesn't interact with [[Vorinclex, Monstrous Raider]] correctly.  If you control Vorinclex and play a saga choosing to start at Chapter 1, I believe you should get triggers for both 1st and 2nd chapters.

Reply:
it's actually supposed to work like that, from the leaked release notes:

![image](https://user-images.githubusercontent.com/29759218/186500595-fc0fad68-7dc0-4992-ad55-ee1dc7c73fe2.png)

Reply:
Huh, that rule seems unintuitive.  Well, this all looks good to me then.

Reply:
it’s weird but I’m guessing it was the easiest way to make the mechanic work in the rules